experiment/httphost/httphost.go

```diff
@@ -0,0 +1,94 @@
+// Package httphost contains the httphost network experiment.
+//
+
+package httphost
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/ooni/probe-engine/experiment/urlgetter"
+	"github.com/ooni/probe-engine/model"
+)
+
+const (
+	testName    = "http_host_header"
```

Comment:
Would you mind renaming the directory and the files `httphostheader`? I feel like this is a nice way to work around the fact that there is a legacy, much-more-complex experiment called `http_host`. Thank!

Reply:
Done.